### ISSUE TITLE
Feature Request: Function to see at the Status LED if the Host does have an Authorized DNS Entry

### ISSUE THREAD

**Issue by Chrisu02** [mRemoteNG/mRemoteNG#2312]
## Expected Behavior
Status RED : Host is not Reachable.
Status Amber: Host is reachable but only with unauthorized adress (nslookup)
Satus Green: Hostname has an Authorized DNS entry

## Current Behavior
Red: Hostname is not available
Green: Hostname is available

## Possible Solution
Adding an Option to add the FQDN domain as Option Field to the settings.
Function to Query the ping if it does have an "Authorized Anwser" or an "unauthorized Anwser"

## Context

The Tool i use at the moment does have this feature, it helps to see if a device is connected directly to the Domain network or via a transparent VPN Service (like Zscaler) to a network, because then the DNS Entry is "unauthorized", this way you then know if you can or can not connect to a device using domain authentification.

**Comment by Kvarkas** [comment]
Interesting idea, will think aout

**Comment by Chrisu02** [comment]
Hello, thanks for thinking about it :)

I noticed that i missunderstood the code, its not the FQDN that is queried for authorized, for me all anwers ar unauthorized.
Its checking the nslookup entry.

so basicly it does ping the Host by its computername, then it does use the ip adress that has been resolved by the ping and does do a nslookup of this IP.

Then it does compare the output computername of the ip nslookup with the originally pinged computername,
If they do match, the Icon is green, if they don't match then the Icon becomes orange.

**Comment by Kvarkas** [comment]
yes i am god on networking i am undestood yours idea :) but in many companies you should know ping could be disabled (for securyti reasons they say) so we cant trust it 100% but as option we may try to add such.

**Comment by robertpopa22** [comment]
**Fix available for testing**

**Commit:** `60f8c5e5` (fork `robertpopa22/mRemoteNG`, branch `main`)
**What changed:** Feature Request: Function to see at the Status LED if the Ho

A beta build (v1.81.0-beta.4) including this fix is available from the fork's Releases page.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
